lib/BaseModeler.js

```diff
@@ -0,0 +1,74 @@
+import inherits from 'inherits';
+
+import Ids from 'ids';
+
+import BaseViewer from './BaseViewer';
+
+
+/**
+ * A base modeler for BPMN 2.0 diagrams.
+ *
+ * Have a look at {@link Modeler} for a bundle that includes actual features.
+ *
+ * @param {Object} [options] configuration options to pass to the viewer
+ * @param {DOMElement} [options.container] the container to render the viewer in, defaults to body.
+ * @param {String|Number} [options.width] the width of the viewer
+ * @param {String|Number} [options.height] the height of the viewer
+ * @param {Object} [options.moddleExtensions] extension packages to provide
+ * @param {Array<didi.Module>} [options.modules] a list of modules to override the default modules
+ * @param {Array<didi.Module>} [options.additionalModules] a list of modules to use with the default modules
```

Comment:
I don't quite get this parameter in BaseModeler. Now that the _modules is empty for BaseModeler/BaseViewer, why would the user pass an additinalModules parameter? Maybe I'm missing a point here though.

Reply:
One could argue that we could get rid of this one.

I did not change and APIs, so that `BaseModeler` and `Modeler` remain interface equivalent.